app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -150,11 +153,17 @@ public void onCreate() {
         PrefUtils.increaseAppStartCount(this);
 
         // Initialize Picasso
+        // When we clone mOkHttpClient, it will use all the same cache and everything.
+        // Only the interceptors will be different.
+        // We shouldn't have the below interceptor in other instances.
+        OkHttpClient picassoClient = mOkHttpClient.clone();

```

Comment:
Picasso client uses an interceptor for plus profile images. When you clone it, it uses the same instance of the internal objects in it. Performance wise it is the same. I found that is Jake Wharton s recommendation for interceptors